### ISSUE TITLE
Falta wireframe de "detalle de producto"

### ISSUE THREAD

**Issue by Piarrot** [1022FSNSANCN02ARED/grupo-04#1]
Hola Grupo! Les estaria faltando para la entrega del sprint 1 el diseño/wireframe del detalle de productos.

Y no termino de entender qué vendria a ser esta pantalla: https://github.com/1022FSNSANCN02ARED/grupo-04/blob/d36648063ba4012a7a11d9dd760af2b493af502e/Wireframe/Productos.png

Pero sospecho que está relacionado a la pantalla faltante ¿Me podrian contar un poquito más?

**Comment by JnOlivera** [comment]
Pablo acá:
https://github.com/1022FSNSANCN02ARED/grupo-04/blob/d36648063ba4012a7a11d9dd760af2b493af502e/Wireframe/Productos.png 
estarían los productos que ofertamos. Podes elegir el que quieras, los mismos van a tener descripciones y tenes el carrito para ir agregando a medida que elegís

**Comment by Piarrot** [comment]
@JnOlivera entiendo, pero la idea es que haya una pantalla de detalle de producto. Sino no puedo ver imagenes extra, comentarios, una descripcion grande que no entraria en una lista.
Esta pantalla que decis de "productos" es como la pagina principal de mercado libre. Cuando tocas en algun producto, te muestra una pagina que es exclusiva de ese producto. Esa es la que está faltando. Habla del envio, de los colores y tamaños u otras propiedades, una descripción bien extensa sobre el producto, multiples imagenes, comentarios, preguntas, reviews, etc etc

No digo que le agreguen todo eso, o que todo eso aplica, pero de verdad consideran que no tienen **NADA** que agregar que lo que van a mostrar en esa pantalla de "productos"?

**Comment by JnOlivera** [comment]
Pablo buen día, se entiende. De hecho es lo que tenemos pensado hacer, cuando selecciones un producto te direccione para ver las características del mismo. El wireframe que subimos es una imagen en la cual no se pueden ver los detalles. Te dejo el link donde lo diseñamos: https://wireframe.cc/pro/pp/e04a6a925612817

**Comment by Piarrot** [comment]
@JnOlivera Claro, de eso se trata este issue. Les falta un wireframe de cómo se vería la **vista de detalle**. Era algo requerido para el sprint 1 y sigue faltando.

**Comment by JoseCourby** [comment]
> @JnOlivera Claro, de eso se trata este issue. Les falta un wireframe de cómo se vería la **vista de detalle**. Era algo requerido para el sprint 1 y sigue faltando.

Pablo, se agregó un screen shot para cumplir con este requerimiento.
